modules/api/php/views/visit/dicom/processes.class.inc

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+/**
+ * PHP Version 7
+ *
+ * @category ApiViews
+ * @package  Loris
+ * @author   Loris Team <loris-dev@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api\Views\Visit\Dicom;
+
+/**
+ * Creates an array representation of a server process.
+ *
+ * @category ApiViews
+ * @package  Loris
+ * @author   Loris Team <loris-dev@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Processes
+{
+    private $_mriuploads;
+
+    /**
+     * Constructor
+     *
+     * @param \LORIS\Data\Models\MRIUploadDTO[] $mriuploads a list or MRIUploadDTO
+     */
+    public function __construct(array $mriuploads)
+    {
+        $this->_mriuploads = $mriuploads;
+    }
+
+    /**
+     * Creates a serializable array of this object's data
+     *
+     * @return array
+     */
+    public function toArray(): array
+    {
+        return [
+            'MriUploads' => array_map(
+                self::_formatMRIUploads(...),
```

Comment:
I am surprised this isn't a syntax error with nothing after the `...`, can you explain what you're trying to do here?

Reply:
I am using the [first class callable syntax](https://www.php.net/manual/en/functions.first_class_callable_syntax.php). If I remember correctly the function used to be passed as a string, but this modern syntax allows to not use a string (which is nice IMO).

Reply:
That makes sense. I was unaware of the syntax and thought it was a splat operator missing a value.